plugins/obs-qsv11/common_utils_windows.cpp

```diff
@@ -172,7 +172,9 @@ double TimeDiffMsec(mfxTime tfinish, mfxTime tstart)
 
 void util_cpuid(int cpuinfo[4], int flags)
 {
+#if !defined(_M_ARM64)
 	return __cpuid(cpuinfo, flags);
+#endif
```

Comment:
Is there an alternative for ARM? Probably not important since QSV would only work a with a dGPU anyway.

Reply:
Right now, there is no direct equivalent for this on ARM.
Reverting these changes as QSV11 is not supported on WOA yet.